src/programs/deadchat.ts

```diff
@@ -1,33 +1,33 @@
-import Tools from "../common/tools";
 import prisma from "../prisma";
 
 import { Command, CommandHandler, DiscordEvent } from "../event-distribution";
-import { Message } from "discord.js";
+import { ChatInputCommandInteraction, TextChannel } from "discord.js";
 
 const thirtyMinutes = 30 * 60 * 1000;
 
-const isDead = async (message: Message): Promise<boolean> => {
-  const lastMessages = (
-    await message.channel.messages.fetch({ limit: 2 })
-  ).values();
+const isDead = async (
+  interaction: ChatInputCommandInteraction
+): Promise<boolean> => {
+  const channel = interaction.channel as TextChannel;
+  const lastMessages = (await channel.messages.fetch({ limit: 2 })).values();
   const lastMessage = [...lastMessages][1];
```

Comment:
This is a sneaky one; previously the bot had to skip the message containing the command when determining whether chat was dead or not. Since the slash command does not count as a message, this is not needed anymore so the code needs to be changed to actually look for the last message (instead of the second to last message as it was previously):

```suggestion
  const lastMessage = (await channel.messages.fetch({ limit: 1 })).first();
  if (!lastMessage) return true;

```

Reply:
Ah, my bad for the oversight.

Reply:
No worries, that's why we do reviews :)